app/core/ingestion/utils.py

```diff
@@ -128,12 +128,29 @@ class Result:
     details: str = ""
 
 
+NameSummary = tuple[str, str]
+
+
+@dataclass
+class MultiplyDefinedEntities:
+    """Used by validation to ensure consistency for MDE's."""
+
+    families: dict[str, NameSummary]  # id -> name, description
+    collections: dict[str, NameSummary]  # id -> name, description
+
+
 @dataclass
 class IngestContext:
     """Context used when processing."""
 
     org_id: int
     results: list[Result]
+    mde: MultiplyDefinedEntities
```

Comment:
`mde` really isn't what I'd call a "well-known acronym", so maybe we can spell it out? Also, I'm not sure that `MultiplyDefinedEntities` really expresses what this thing is for. Would something like `ValidationLookup` no express more what this class is for?